smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -72,7 +72,7 @@ def authPage() {
 		log.debug "have LIFX access token"
 
 		def options = locationOptions() ?: []
-		def count = options.size()
+		def count = options?.size() > 0 ? options.size()  : 0
```

Comment:
Juan, are you able to reproduce the original problem and verify that this change fixes it? Now that I'm looking at the whole thing I don't see how `count` could ever actually be `null`. If the call to `locationOptions()` on line 74 returns null then `options` is set to an empty list, the size of which would be 0.